src/signed_data.rs

```diff
@@ -296,6 +296,26 @@ pub static RSA_PKCS1_3072_8192_SHA384: SignatureAlgorithm = SignatureAlgorithm {
     verification_alg: &signature::RSA_PKCS1_3072_8192_SHA384,
 };
 
+/// RSA PSS signatures using SHA-256 for keys of 2048-8192 bits.
+pub static RSA_PSS_2048_8192_SHA256: SignatureAlgorithm = SignatureAlgorithm {
```

Comment:
I've done, that albeit referencing https://tools.ietf.org/html/rfc4055#section-1.2 because it provides an explanation of what's going on.